docs/pages/development/build.md

```diff
@@ -57,7 +57,7 @@ You can register an iOS device and install a provisioning profile with the follo
 Once you've registered all iOS devices you'll want to run your development build on, you can run the following to create a build ready for internal distribution:
 <Terminal cmd={["$ eas build --profile development --platform ios"]} />
 
-To share the build with your team, direct them to the build page on https://expo.dev. There, they'll be able to download the app directly on their device.
+To share the build with your team, direct them to the build page on https://expo.dev . There, they'll be able to download the app directly on their device.
```

Comment:
```suggestion
To share the build with your team, direct them to the build page in [Expo dashboard](https://expo.dev/accounts/[account]/projects/[project]/builds). There, they'll be able to download the app directly on their device.
```

Let's use direct link to the builds page.

Reply:
@Simek Thanks a bunch, I'll update it using [](), which helps to separate the period with the url.

Reply:
Can you also update the URL of the link? 

Instead of linking to the Expo homepage we can redirect users to the build page of their chosen project via Link Resolver view.

Reply:
Sure, could you please help me understand what is the build page of the chosen project via Link Resolver view.

Reply:
You can see the proposed link in my first review suggestion. Instead of using:
* https://expo.dev

We can use URL like this:
* https://expo.dev/accounts/[account]/projects/[project]/builds

Reply:
Gotcha, Thanks I missed it